features/reports_by_day_metrics.feature

```diff
@@ -0,0 +1,207 @@
+@api @metrics
+Feature: Reports by day metrics
+
+  Users should be able to retrieve the number of reports by day of an organization.
+  The users can filter the by project and/or by user.
+  Finally, the users can retrieve between 1 and 120 days of data.
+
+  The following result is provided:
+  - runsCount for each day
+  - date for each day
+
+
+  Background:
+    Given private organization Rebel Alliance exists
+    And user hsolo who is a member of Rebel Alliance exists
+    And user lskywalker who is a member of Rebel Alliance exists
+    And project X-Wing exists within organization Rebel Alliance
+    And test result report report 1 was generated 2 days ago for organization Rebel Alliance
+    And test payload payload 1 sent by lskywalker for version 1.0.0 of project X-Wing was used to generate report report 1
+    And test payload payload 2 sent by hsolo for version 1.0.0 of project X-Wing was used to generate report report 1
+    And test result report report 2 was generated 2 days ago for organization Rebel Alliance
+    And test payload payload 3 sent by lskywalker for version 1.0.0 of project X-Wing was used to generate report report 2
+    And test result report report 3 was generated 1 days ago for organization Rebel Alliance
```

Comment:
I wouldn't constraint the regexp, as it's quite un-Rails-like (e.g. Rails accepts `1.days.ago` too), and the simpler the regexp the better.